pkg/webui/account/store/actions/collaborators.js

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
```

Comment:
The `collaborator` store modules are now duplicated between console and account app. It should be moved to a top-level module into `/lib/store/*/collaborator.js`

Reply:
Is moving the `collaborators` middleware logic to `/lib/store` good to do as well? I mean, one of the imports in that file is `tts api` -  if the requests refer to the account app it should be imported from `@account/api`, otherwise from `@console/api`, if it's in the global scope how are we going to differentiate?

Reply:
Yes, good catch. You can use a generator function that will take the tts instance and then builds the logics accordingly.

```
// webui/lib/store/middleware/collaborators.js

export default createCollaboratorLogics = tts => {
  const getCollaboratorLogic = createRequestLogic({ ... })
  const getCollaboratorsLogic = createRequestLogic({ ... })
  ...
  
  return [getCollaboratorLogic, getCollaboratorsLogic, ...]
}

// webui/console/store/middleware/logics/collaborators.js

export default createCollaboratorLogics(tts)
```